tests/integration/server/api_users_tests.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+"""Validating User Account Creation API Endpoint Test"""
+
+import requests
+
+
+def create_test_user(session):
+    create_user_input = {
+        "name": "Sri Nuthal",
+        "username": "nuthalapati",
+        "password": "123456",
+        "email": "sri@gmail.com",
+    }
+    session.post("http://rbac-server:8000/api/users", json=create_user_input)
+
+
+def test_valid_username():
+    """ Testing the already existing Username scenario. """
```

Comment:
You should clarify in your doc strings that you are testing the creation of two users with different usernames here.